client/app/shared/modals/model-edit/model-edit.component.ts

```diff
@@ -49,4 +49,8 @@ export class ModelEdit {
     return ['text', 'time'].some(type => type === checkedType)
   }
 
+  trackByField (field) {
+    return field
```

Comment:
The `trackBy` function will get an index in the first parameter and the item in the second parameter, the index shouldn't be the output of the `trackBy` function, use something more relevant to the field like `field.display`